app/serializers/alchemy/json_api/element_serializer.rb

```diff
@@ -17,8 +17,7 @@ class ElementSerializer
       end
 
       belongs_to :parent_element, record_type: :element, serializer: self
-
-      belongs_to :page, record_type: :page, serializer: ::Alchemy::JsonApi::PageSerializer
+      belongs_to :page, through: :page_version, record_type: :page, serializer: ::Alchemy::JsonApi::PageSerializer
```

Comment:
Do we need this? I seem to remember we don't want to access an element's page from the element, and if we do want that, the calling code should know about the page.

Reply:
There's a subsequent commit that removes the line, maybe this can be rebased out.

Reply:
Done